src/utils/gameTimer.ts

```diff
@@ -0,0 +1,21 @@
+import type { GameStartType } from "..";
+import { TimerType } from "..";
+import { format } from "date-fns";
+
+export function frameToGameTimer(frame: number, gameStart: GameStartType): string {
+  const timerType = gameStart.timerType;
+
+  if (timerType == TimerType.DECREASING) {
+    const centiseconds = Math.ceil((((60 - (frame % 60)) % 60) * 99) / 59);
+    const date = new Date(0, 0, 0, 0, 0, (gameStart.startingTimerSeconds as number) - frame / 60, centiseconds * 10);
+    return format(date, "mm:ss.SS");
+  }
+
+  if (timerType == TimerType.INCREASING) {
+    const centiseconds = Math.floor(((frame % 60) * 99) / 59);
+    const date = new Date(0, 0, 0, 0, 0, frame / 60, centiseconds * 10);
+    return format(date, "mm:ss.SS");
+  }
+
+  return "";
```

Comment:
Do you think we should probably return something else here instead. Maybe like "Infinite" or "Endless" or something?

Reply:
Originally I wrote this to get the exact display of the timer and added the empty string to for full coverage - if there isn't one then there is nothing. It could potentially cause confusion but I think that scenario would be really rare. Of the suggestions, I like "Infinite" but it really does not matter to me.